package.json

```diff
@@ -14,6 +14,7 @@
     "webpack-merge": "^4.2.2"
   },
   "scripts": {
+    "start": "bin/webpack-dev-server & rails s",
```

Comment:
this is more `start:dev` right ? `start` as in production would be `bundle exec rails server -b 0.0.0.0` if I'm not wrong?

Reply:
That's a good point, I'll add a start:production script too. From what I read bundle exec rails server and rails server are the same, apparently the rails command already has integrated support to check which version it should run, maybe I'm missing something?

Reply:
yeah they are the same :)